### PR TITLE
Practice5 pull request

### PR DESCRIPTION
**Pull request by andrew-budnitskiy**
This is a pull request for **fifth** practice
Andrew Budnitskiy
12.08.2021

### CODE REVIEW

File: OI_Practice/OI_Practice/Maps/Map/MapViewController.swift [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+            // not need to show callout - let off it
+            pinView.canShowCallout = false
+            return pinView
+        }
+
+    // occures when we tap on the map
+    // removes old and put a new pin to the map surface
+    override func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
+
```
Comment (andrew-budnitskiy) [inline OI_Practice/OI_Practice/Maps/Map/MapViewController.swift:168]: I have a trouble. It was used MKMarkerAnnotationView for present marker, but click on map marker, created by this annotation's view fires not always quiet as necessary - click on map event fires even when i tap on marker often.
I use MKAnnotationView's subview, created from xib too, but it had no effect.
How can i create a marker with title, click on that doesn't fire click on map?

Comment (andrew-budnitskiy) [reply OI_Practice/OI_Practice/Maps/Map/MapViewController.swift]: fixed

File: .gitignore [hunk 2]
```diff
@@ -85,9 +95,16 @@ fastlane/Preview.html
 fastlane/screenshots/**/*.png
 fastlane/test_output
 
+# Ignore all other practices
```
Comment (vasilevkin) [inline .gitignore:98]: do not add this in gitignore

**Review by vasilevkin — CHANGES_REQUESTED** [review]
this pr should contain changes related to practice 5 only.
please check carefully what files and commits are included.

you can create a new pr instead of this one, if you like

File: .gitignore (hunk 2 above)
Comment (andrew-budnitskiy) [reply .gitignore:98]: fixed

**Comment by andrew-budnitskiy** [comment]
> this pr should contain changes related to practice 5 only.
> please check carefully what files and commits are included.
> 
> you can create a new pr instead of this one, if you like

fixed

**Pull request closed without merging**